components/blitz/src/omero/model/ElectricPotentialI.java

```diff
@@ -609,35 +620,28 @@ public ElectricPotentialI(double d, ome.model.enums.UnitsElectricPotential ul) {
     *
     * @param target String representation of the CODE enum
     */
-    public ElectricPotentialI(ElectricPotential value, String target) {
+    public ElectricPotentialI(ElectricPotential value, String target) throws BigResult {
        String source = value.getUnit().toString();
        if (target.equals(source)) {
            setValue(value.getValue());
            setUnit(value.getUnit());
         } else {
-            double[][] coeffs = conversions.get(source + ":" + target);
-            if (coeffs == null) {
+            Conversion conversion = conversions.get(source + ":" + target);
```

Comment:
If we have an array of many same-unit values to convert, it's a pity if we have to do this string concatenation and map lookup for every one in converting the lot, but I don't see an easy way around that and it may not be much cost compared to other construction costs anyway.